src/CMakeLists.txt

```
@@ -45,6 +44,13 @@ set(lvarray_sources
     totalview/tv_data_display.c 
     )
 
+if(ENABLE_PYTHON)
+    set(lvarray_headers
+        python/numpyConversion.hpp
+        python/numpyArrayView.hpp
+        python/numpySortedArrayView.hpp
+        )
```

Shouldn't this be
```suggestion
    list( APPEND lvarray_headers
          python/numpyConversion.hpp
          python/numpyArrayView.hpp
          python/numpySortedArrayView.hpp
          )
```

?